cluster/v1beta1/helpers.go

```diff
@@ -82,3 +90,29 @@ func BuildClusterSelector(clusterSet *ManagedClusterSet) (labels.Selector, error
 		return nil, fmt.Errorf("selectorType is not right: %s", clusterSet.Spec.ClusterSelector.SelectorType)
 	}
 }
+
+// GetValidManagedClusterSetBindings returns all bindings found in the namespace.
+func GetValidManagedClusterSetBindings(namespace string,
+	clusterSetBindingsGetter ManagedClusterSetBindingsGetter,
+	clusterSetsGetter ManagedClusterSetsGetter) ([]*ManagedClusterSetBinding, error) {
+	// get all clusterset bindings under the namespace
+	bindings, err := clusterSetBindingsGetter.List(namespace, labels.Everything())
```

Comment:
hrm, should we have a  binding status to reflect whether there is valid clusterset? This could be useful since an operator would not need permission to list clusterset.

FYI @elgnay @ldpliu

Reply:
There is an issue to trace this: https://github.com/stolostron/backlog/issues/21580#issuecomment-1094625671